### ISSUE TITLE
can't reclaim team ownership if owner resets account

### ISSUE THREAD

**Issue by wxl** [keybase/client#9677]
STEPS TO REPRODUCE
====================
 1. User creates team
 2. User adds admins
 3. User resets account
 4. Email is generated to admins telling them that the user was "kicked out" and is "cryptographically" different but suggests adding them back

EXPECTED RESULTS
==================
You can just follow the instructions to add the user back.

ACTUAL RESULTS
================
 * Cannot add user back, either with suggested `keybase team add-member` or under the "Invites" tab in the team (the user had made a request).
   * CLI error: `▶ ERROR cannot post link (precheck): appending 6->7: non-owner cannot demote owners`
   * UI error: `user`*`"username"`*`is not a member of team`*`"teamname"`*
 * Admins cannot add owners.
 * Admins cannot demote owners.
 * Admins cannot remove owners.

AFFECTED VERSIONS
===================
 * Android 1.0.34-20171027214850+c5ade4b91
 * Ubuntu 1.0.36-20171122003226.f68b10ef8

I'm posting this on behalf of @vikingredwolf who is the user affected. I'm one of the other admins on a team we're both in, or not, depending on how you interpret the above. ;)

**Comment by maxtaco** [comment]
cc @mlsteele on this...

**Comment by mlsteele** [comment]
Oh sugar. Thank for this extremely clear report! This is an oversight. Will need a client release once fixed.

**Comment by maxtaco** [comment]
@mlsteele does this mean we need to implement admin promotion?

**Comment by mlsteele** [comment]
We should allow admins to remove owners who have reset. Or replace them with higher seqnos. (Or leave the reset owner concurrent with the new UV but I don’t like this option much).

**Comment by maxtaco** [comment]
Yeah, I agree, but it could be no group operation is going to work on the server is the number of owners is 0 (and we do an invariant check after the DB update to make sure it never drops below 1).  Uggggg, account resets!
